sklearn/linear_model/_ridge.py

```diff
@@ -772,6 +775,22 @@ def _ridge_regression(
 
 
 class _BaseRidge(LinearModel, metaclass=ABCMeta):
+
+
+    _parameter_constraints = {
+        "alpha": [Interval(Real, 1, None, closed="left"), float, np.ndarray],
+        "fit_intercept": [bool],
+        "normalize": [StrOptions({"deprecated"}), bool],
```

Comment:
this should be `Hidden(StrOptions({"deprecated"}))` but needs this PR https://github.com/scikit-learn/scikit-learn/pull/23558 merged before